_pytest/config.py

```diff
@@ -917,7 +917,10 @@ def parse(self, args):
         self.hook.pytest_cmdline_preparse(config=self, args=args)
         args = self._parser.parse_setoption(args, self.option)
         if not args:
-            args.append(os.getcwd())
+            if os.getcwd() == self.rootdir:
+                args = self.getini('testpaths')
+            if not args:
+                args = [os.getcwd()]
         self.args = args
```

Comment:
minor nit. os.getcwd() should be assigned to a var and then this var used instead of potentially calling it twice IMHO.